tests/wasmbus.rs

```diff
@@ -47,13 +47,14 @@ async fn free_port() -> anyhow::Result<u16> {
 }
 
 async fn assert_start_actor(
-    ctl_client: &wasmcloud_control_interface::Client,
+    ctl_client: &wasmcloud_control_interface::Client<DirectKvStore>,
     _nats_client: &async_nats::Client, // TODO: This should be exposed by `wasmcloud_control_interface::Client`
     _lattice_prefix: &str,
     host_key: &KeyPair,
     url: impl AsRef<str>,
     count: u16,
 ) -> anyhow::Result<()> {
+    #[allow(deprecated)]
```

Comment:
I feel like there should be a comment here about what is deprecated and when we'll remove it